bundler/spec/other/major_deprecation_spec.rb

```diff
@@ -647,4 +647,29 @@
       end
     end
   end
+
+  context "when user set install parameter of inline gem to true" do
+    def script(code, options = {})
+      requires = ["#{entrypoint}/inline"]
+      requires.unshift "#{spec_dir}/support/artifice/" + options.delete(:artifice) if options.key?(:artifice)
+      requires = requires.map {|r| "require '#{r}'" }.join("\n")
+      ruby("#{requires}\n\n" + code, options)
+    end
+
+    it "shows platform warnings", :bundler => "< 3"do
+      script <<-RUBY
```

Comment:
I _think_ for testing this you can use the `ruby` helper and add a `require "bundler/inline"` on top of the script, and you no longer need the `script` helper above?

```suggestion
      ruby <<-RUBY
        require "bundler/inline"
        
```

Reply:
I'll try this and let you know.

Reply:
I'm getting this error: `No such plugin: ruby`

Reply:
Are you using `asdf`? Can you run `unset RUBYLIB` and try again?

Reply:
Still getting same error

Reply:
Can you still push the change, just to confirm it's an issue with asdf? I don't understand why one works, and the other doesn't to be honest.

Reply:
Sure.